api/src/test/java/org/apache/iceberg/expressions/TestEvaluator.java

```diff
@@ -220,6 +222,27 @@ public void testNotEqual() {
 
   }
 
+  @Test
+  public void testStartsWith() {
+    StructType struct = StructType.of(required(24, "s", Types.StringType.get()));
+    Evaluator evaluator = new Evaluator(struct, startsWith("s", "abc"));
+    Assert.assertTrue("abc startsWith abc should be true", evaluator.eval(TestHelpers.Row.of("abc")));
```

Comment:
Should we add a test with the pattern present, but not a the start?
```suggestion
    Assert.assertTrue("abc startsWith abc should be true", evaluator.eval(TestHelpers.Row.of("abc")));
    Assert.assertFalse("xabc startsWith abc should be false", evaluator.eval(TestHelpers.Row.of("xabc")));
```

Reply:
Good call. Will add.

Reply:
Added. Passes as expected.